dbms/src/Flash/Mpp/MPPTaskManager.cpp

```diff
@@ -153,6 +154,7 @@ void MPPTaskManager::abortMPPQuery(UInt64 query_id, const String & reason, Abort
         /// just to double check the query still exists
         if (it != mpp_query_map.end())
             mpp_query_map.erase(it);
```

Comment:
or
```
if (it != mpp_query_map.end())
{
    mpp_query_map.erase(it);
    GET_METRIC(tiflash_mpp_task_manager, type_mpp_query_count).Set(mpp_query_map.size());
}         
```
?

Reply:
It's ok since it will be removed in the next pr.